src/lib/blitsfileconverter/blitsfileconverter.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2023 Comcast Cable Communications Management, LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export default (source) => {
+  const { template, script } = parseBlitsFile(source)
+  return injectTemplate(script, template)
+}
+
+const parseBlitsFile = (source) => {
+  const templateMatch = source.match(/<template>([\s\S]*?)<\/template>/)
+  const scriptMatch = source.match(/<script>([\s\S]*?)<\/script>/)
```

Comment:
script match regular expression is not handled for typescript  support. 
`<script lang="ts"> </script>`

Reply:
Thanks @suresh-gangumalla . I have fixed it